src/views/css-styles/Checked.jsx

```
@@ -0,0 +1,46 @@
+          </div>
+        </Card>
+        <Card>
+          <h2>Checkbox</h2>
+          <div className="flex flex-col gap-2">
+            <div>
+              <input type="checkbox" id="check" className="checkbox-1 peer"/>
+              <label for="check" className="peer-[.checkbox-1]:peer-checked:text-red-500">Check 1</label>
+            </div>
+            <div>
+              <input type="checkbox" id="check2" className="checkbox-2 peer"/>
+              <label for="check2" className="peer-[.checkbox-2]:peer-checked:text-red-500">Check 2</label>
```

ESLint

[error] Unknown property 'for' found, use 'htmlFor' instead

(react/no-unknown-property)